### PR TITLE
Change keymap for toogle-output

### PR DESCRIPTION
**Pull request by carusogabriel**
As suggested in #12 and I think it's pretty helpful, let's make it: `Esc` to toggle the output panel (without losing the real `Esc` funcionallity in the Command Palette) :o:

### CODE REVIEW

**Comment by itmecho** [comment]
Hi @gabriel-caruso 

Would it be possible to leave the old command in there as well to make sure we're being as flexible as possible? That way, anyone who is using it and is used to the old way won't be affected.

**Comment by carusogabriel** [comment]
@Synapse791 Do you mean bring back `alt-shift-x`?

File: keymaps/atom-phpunit.json [hunk 1]
```diff
@@ -3,7 +3,9 @@
     "alt-shift-t": "atom-phpunit:run-test",
     "alt-shift-c": "atom-phpunit:run-class",
     "alt-shift-s": "atom-phpunit:run-suite",
-    "alt-shift-r": "atom-phpunit:run-last-test",
-    "alt-shift-x": "atom-phpunit:toggle-output"
+    "alt-shift-r": "atom-phpunit:run-last-test"    
+  },
+  "atom-workspace atom-text-editor:not([mini])": {
+    "escape': 'atom-phpunit:toggle-output"
```
Comment (claytonrcarter) [inline keymaps/atom-phpunit.json:9]: The quotes are wrong here: all of the quotes should be double quotes ("), with no single quotes (')

**Comment by claytonrcarter** [comment]
@gabriel-caruso thank you for getting this started!

Aside from the quotes being wrong in the keymap (see https://github.com/Synapse791/atom-phpunit/pull/17/files#r151702599), I don't like that this opens the output panel when I hit `Escape`. I think `Escape` should only hide the output panel when it's visible, not open it when it's hidden. Also, I agree that `alt-shift-x` should be kept to toggle the output panel.

**Comment by claytonrcarter** [comment]
Actually, after playing with this a little more, this keymap breaks all sorts of stuff. If I have multiple cursors, I can no long `escape` to cancel them and if I have an autocomplete popup visible, hitting `escape` doesn't dismiss it. In both cases, the output panel is displayed (or hidden), but the cursors and/or popup remain visible.

**Pull request closed without merging**